tt_eager/tt_dnn/op_library/groupnorm/kernels/compute/groupnorm_sharded_v2.cpp

```diff
@@ -196,7 +180,11 @@ void MAIN {
                     for (uint32_t w = 0; w < subblock_w; ++w) {
                         uint32_t index = w + index_subblock_w_offset + index_h_offset;
                         uint32_t index_mask = w + index_subblock_w_offset;
+                        #ifdef TILIZE_IN
                         mul_tiles(cb_in, cb_input_mask, index, index_mask, w);
+                        #else
+                        mul_tiles(cb_in0, cb_input_mask, index, index_mask, w);
+                        #endif
```

Comment:
I can't seem to understand what is the difference between `cb_in` and `cb_in0`
Can you please educate me a little here?

Reply:
these are just different cb index, and those cbs are in different L1 locations,